IBMa/equal-access
# fix(engine): turn off the "emoticons_alt_exists" and "asciiart_alt_exists" rules

Rule bug: emoticons_alt_exists, asciiart_alt_exists

### This PR is related to the following issue(s): 
- 
#805 

### Additional information can be found here: 
- 


### Testing reference: 
- 
1. run the test/v2/checker/accessibility/rules/asciiart_alt_exists_ruleunit/AsciiArt-detectInvalid.html, the NR: "Verify that ASCII art has a text alternative" shouldn't show up.
2. run test/v2/checker/accessibility/rules/emoticons_alt_exists_ruleunit/Text-hasEmoticons.html, the NR: "Verify that ASCII art has a text alternative" shouldn't show up.

### I have conducted the following for this PR: 
- [x ] I validated this fix in my local env
- [x ] I provided details for testing
- [x ] I understand that the title of this PR will be used for the next release notes.
